bin/packages/get-babel-config.js

```diff
@@ -11,6 +11,8 @@ module.exports = ( environment = '', file ) => {
 			name: `WP_BUILD_${ environment.toUpperCase() }`,
 		},
 	};
+	// Add `@wordpress/polyfill` import where needed.
```

Comment:
This comment is out of date, the magic comments are added by babel, not the import.

Reply:
Well spotted! Fixed.